src/tests/unit/tests/DetailsView/components/assessment-instance-table.test.tsx

```diff
@@ -108,9 +108,13 @@ describe('AssessmentInstanceTableTest', () => {
 
         const expected = (
             <div>
-                <ActionButton iconProps={{ iconName: 'skypeCheck' }} onClick={testObject.getOnPassUnmarkedInstances()} disabled={true}>
+                <InsightsCommandButton
+                    iconProps={{ iconName: 'skypeCheck' }}
+                    onClick={testObject.getOnPassUnmarkedInstances()}
+                    disabled={true}
+                >
                     Pass unmarked instances
-                </ActionButton>
+                </InsightsCommandButton>
```

Comment:
is it worth it to use this opportunity to convert this tests to use enzyme and snapshots?
The `TestableAssessmentInstanceTable` always gives me an itch.

Reply:
I've done this, but the change to the tests is very substantial, so I'd rather put it in its own PR (I'll send that out after this one is merged)